docs/docs/how-to/local-development/environment-variables.md

```diff
@@ -122,83 +40,23 @@ module.exports = {
 }
 ```
 
-## Reserved Environment Variables:
-
-> You can not override certain environment variables as some are used internally
-> for optimizations during build, such as:
-
-- `NODE_ENV`
-- `PUBLIC_DIR`
-
-Gatsby also allows you to specify another environment variable when running the local development server (e.g. `npm run develop`):
-
-- `ENABLE_GATSBY_REFRESH_ENDPOINT`
-
-This allows you to refresh your sourced content. See [Refreshing Content](/docs/refreshing-content/).
-
-## Build Variables
-
-Gatsby uses additional environment variables in the build step to fine-tune the outcome of a build. You may find these helpful for more advanced configurations, such as using [CI/CD](https://en.wikipedia.org/wiki/CI/CD) to deploy a Gatsby site.
-
-For example, you can set `CI=true` as an environment variable to allow Gatsby's build script to tailor the terminal output to an automated deployment environment. Some CI/CD tooling may already set this environment variable. This is useful for limiting the verbosity of the build output for [dumb terminals](https://en.wikipedia.org/wiki/Computer_terminal#Dumb_terminals), such as terminal in progress animations.
-
-Gatsby detects an optimal level of parallelism for the render phase of `gatsby build` based on the reported number of physical CPUs. For builds that are run in virtual environments, you may need to adjust the number of worker parallelism with the `GATSBY_CPU_COUNT` environment variable. See [Multi-core builds](/docs/multi-core-builds/).
-
-## Additional Environments (Staging, Test, etc.)
-
-As noted above `NODE_ENV` is a reserved environment variable in Gatsby as it is needed by the build system to make key optimizations when compiling React and other modules. For this reason it is necessary to make use of a secondary environment variable for additional environment support, and manually make the environment variables available to the client-side code.
-
-You can define your own OS Env Var to track the active environment, and then to locate the relevant Project Env Vars to load. Gatsby itself will not do anything with that OS Env Var, but you can use it in `gatsby-config.js`.
-
-Specifically, you can use `dotenv` and your individual OS Env Var to locate the `.env.myCustomEnvironment` file, and then use `module.exports` to store those Project Env Vars somewhere that the client-side JavaScript can access the values (via GraphQL queries).
-
-### Google Analytics env var example
-
-If you would like to add a `staging` environment with a custom Google Analytics Tracking ID, and a dedicated `apiUrl` you can add `.env.staging` at the root of your project. In order to do so, use the following modification to your `gatsby-config.js`:
-
-```text:title=.env.staging
-GA_TRACKING_ID="UA-1234567890"
-API_URL="http://foo.bar"
-```
-
-```javascript:title=gatsby-config.js
-const activeEnv =
-  process.env.GATSBY_ACTIVE_ENV || process.env.NODE_ENV || "development"
+## Accessing Environment Variables in the browser.
```

Comment:
I'm missing an example here on how to use `GATSBY_API_URL` in the frontend code

Reply:
added